### PR TITLE
Add a (simple) PDF profile 

### PR DESCRIPTION
**Pull request by llemeurfr**
Because we need to have a PDF profile released asap, Here is a simple proposal, with a text aligned with the other profiles and no introduction of a new property, as in https://github.com/readium/webpub-manifest/pull/88. 

If we agree about the addition of a `start` property (which means that every Readium toolkits will be ready to implement it), we'll update this profile accordingly.

### CODE REVIEW

File: profiles/pdf.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  
+  "resources": [
+    {"rel": "cover", "href": "http://example.org/cover.jpeg", "type": "image/jpeg", "height": 600, "width": 300}
+  ]
+}
+```
+
+## Introduction
+
+The goal of this specification is to provide a profile of the [Readium Web Publication Manifest](https://readium.org/webpub-manifest) dedicated to pdf documents.
```
Comment (HadrienGardeur) [inline profiles/pdf.md:50]: Should be "PDF documents" instead of lowercase pdf.

File: profiles/pdf.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+It is mostly used as a basis for the specification of a method for protecting PDF files using LCP, i.e. [LCP for PDF](https://readium.org/lcp-specs/notes/lcp-for-pdf.html). 
+
+This profile relies on:
+
+* a declaration of [conformance with this Profile](#1-declaring-conformance-with-the-pdf-profile),
+* some [restrictions on the resources of the reading order](#2-restrictions-on-the-resources-of-the-reading-order),
```
Comment (HadrienGardeur) [inline profiles/pdf.md:57]: It's a single restriction currently.

File: profiles/pdf.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## 1. Declaring conformance with the PDF Profile
+
+To declare that it conforms to the PDF Profile, a Readium Web Publication Manifest <strong class="rfc">must</strong> include a `conformsTo` key in its `metadata` section, with `https://readium.org/webpub-manifest/profiles/pdf` as value.
```
Comment (HadrienGardeur) [inline profiles/pdf.md:61]: This should be modified to say "with `https://readium.org/webpub-manifest/profiles/pdf` as one of its values."

File: profiles/pdf.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+## 2. Restrictions on resources in the reading order
+
+In addition to the normal requirements of a `readingOrder`, all Link Objects <strong class="rfc">must</strong> point strictly to PDF resources, with no fragment identifier. 
+
+> **Note**
```
Comment (HadrienGardeur) [inline profiles/pdf.md:67]: I would drop the note, this does not belong in a spec.

**Review by HadrienGardeur — COMMENTED** [review]
Not much to say overall, just minor tweaks.

File: profiles/pdf.md (hunk 2 above)
Comment (llemeurfr) [reply profiles/pdf.md:57]: done

File: profiles/pdf.md (hunk 1 above)
Comment (llemeurfr) [reply profiles/pdf.md:50]: done

File: profiles/pdf.md (hunk 4 above)
Comment (llemeurfr) [reply profiles/pdf.md:67]: ok.

**Pull request merged**